README.md

```diff
@@ -42,8 +42,6 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230603891-61f95ca6-671f-44c7-82cf-1aeb1f5651ee.png)

**[OPTIONAL]**

- [ ] It would be great if you add your logo at the top of your readme file to make yourself a branch in the tech industry to catch the eyes of recruiters 👍